navigation.yaml

```
@@ -1018,6 +1018,9 @@ core:
       path: /core/docs
       persist: True
 
+    - title: Success Stories
```

```suggestion
    - title: Success stories
```

Or should it be 'Customer stories'  I don't mind - @wgx ?

---

@pmahnke I like "Success stories" better - has a bit more personality than just customer stories...